lib/unhangout-routes.js

```diff
@@ -491,18 +549,51 @@ module.exports = {
             showEventOverview(req, res, "admin.ejs");
         });
 
-        app.get("/myevents/", ensureAuthenticated, function(req, res) {
+        app.get("/myevents/", ensureAuthenticated, utils.ensureAdmin,function(req, res) {
             showEventOverview(req, res, "myevents.ejs");
         });
 
+        app.post("/myevents/", ensureAuthenticated, utils.ensureAdmin, function(req, res) {
+
+            var eventID = req.body.eventId; 
+
+            var event = db.events.get(req.body.eventId);
+
+            if (!event) {
+                return res.send(400, "Missing event.");
```

Comment:
Mostly in our code for missing events, we are using the status code 400. I followed the same here. If you strongly feel that for this case, it should be 404, I will make the change.